servers/rendering/renderer_compositor.h

```diff
@@ -107,6 +108,7 @@ class RendererCompositor {
 	static bool is_low_end() { return low_end; };
 	virtual bool is_xr_enabled() const;
 
+	static RendererCompositor *get_default_singleton() { return singleton; }
```

Comment:
This can just be called `get_singleton`. As it's static it's clear from calling `RendererCompositor::get_singleton` what you're getting, even if get_singleton is also implemented on subclasses.

Reply:
It's not possible to define it as `get_singleton` if you want to define a new singleton in the derived class.

Reply:
Hmm, I do this in `RenderForwardClustered` and `RenderForwardMobile` which both inherit from `RendererSceneRenderRD` and all 3 have their own singleton member and `get_singleton` method. It's fairly important here because the likelihood of code being written that is shared between both render implementations yet calls the wrong `get_singleton` is relatively high.

Reply:
Ah, I thought it's not possible for static methods in C++...

Reply:
*Fixed* like you said